src/sentry/api/bases/project.py

```diff
@@ -32,37 +28,9 @@ def has_object_permission(self, request: Request, view, project):
 
         if not result:
             return result
-        if project.teams.exists():
-            return any(
-                has_team_permission(request, team, self.scope_map) for team in project.teams.all()
-            )
-        elif is_system_auth(request.auth):
-            return True
-        elif request.user and request.user.is_authenticated:
-            # this is only for team-less projects
-            if is_active_superuser(request):
-                return True
-            elif request.user.is_sentry_app:
-                return SentryApp.objects.check_project_permission_for_sentry_app_user(
-                    request.user, project
-                )
```

Comment:
I looked at `sentry.auth.access` and didn't see any project level installation checks, should we have those still? Right now the access methods would allow a sentry app request to a project that the sentry app wasn't enabled for.

Reply:
if you check out `check_project_permission_for_sentry_app_user` (deleted below), it simply checks that the project is in the organization of the installation, so there wasn't a project specific check for sentry_apps. The behavior is confirmed in `tests/sentry/api/bases/test_project.py` with the sentry_app tests